### DISCUSSION TITLE
How to insert "use system python" option through file writing?

### DISCUSSION THREAD

**Discussion by aaronliu0130** [chaiNNer-org/chaiNNer#2328] [Q&A] [answered]
Or, how do I use scripts to make chaiNNer use system python even if integrated python hasn't been downloaded yet?

**Comment by joeyballentine** [comment, marked as answer]
Navigate to `C:\Users\[your user]\AppData\Roaming\chaiNNer\settings`, create a file (if it doesnt exist) called `use-system-python` (no extension), open it in a text editor (notepad is fine), type `true` into the file, and save it.

Then it should use your system python.

Let me know if you run into any other issues

**Reply by aaronliu0130** [reply]
So it’s the same folder but under `~/.config` in Linux?

**Reply by joeyballentine** [reply]
I'm not super familiar with Linux's filesystem but if that's where the equivalent of appdata is, then yes. I'll try to confirm later

**Reply by aaronliu0130** [reply]
Yep it is, can confirm!
